pkg/oc/clusterup/run_self_hosted.go

```diff
@@ -529,8 +529,10 @@ func (c *ClusterUpConfig) startKubelet(out io.Writer, masterConfigDir, nodeConfi
 	} else if currentUser, err := user.Current(); err == nil {
 		var cfgPath string
 		cfgPath = filepath.Join(currentUser.HomeDir, ".docker", "config.json")
-		container.ContainerBinds = append(container.ContainerBinds, fmt.Sprintf("%s:/root/.docker/config.json", cfgPath))
-		glog.Infof("docker config path %s", cfgPath)
+		// First check if config file exist on the host before add it to the mounts.
+		if _, err := os.Stat(cfgPath); os.IsExist(err) {
```

Comment:
is this right?

shouldnt it check `if _, err := os.Stat(path); err == nil {`

Reply:
Yes